Cargo.toml

```diff
@@ -20,14 +20,14 @@ members = [
 [dependencies]
 ferrischat_webserver = { path = "ferrischat_webserver", version = "0.1.0" }
 ferrischat_config = { path = "ferrischat_config", version = "0.1.0" }
-actix-web = { git = "https://github.com/actix/actix-web/" }
+actix-web = { git = "https://github.com/actix/actix-web/", version = "4.0.0-beta.10"}
```

Comment:
i don't want to lock it on beta.10, since later versions might add some nice new stuff